components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -563,6 +563,7 @@ def getJSToNativeConversionInfo(type, descriptorProvider, failureCode=None,
                                 isMember=False,
                                 isArgument=False,
                                 isAutoRooted=False,
+                                needsRootingInto=False,
```

Comment:
To briefly summarize from my [other comment](https://github.com/servo/servo/pull/20205/files/4098828954a27e4c3ce12e23ac5caf724e67309a#r172486002), the current typed array conversion itself requires prior stack root, whereas argument conversions with `isAutoRooted` only roots the type after the conversion - to avoid this, it'd need adding unrooted object typed array conversion API and/or supporting rooting those with CustomAutoRooter